ci/render_pipeline_run.py

```diff
@@ -295,6 +305,7 @@ def main():
     parser.add_argument('--git-url', default='https://github.com/gardenlinux/gardenlinux.git')
     parser.add_argument('--flavour-set', default='all')
     parser.add_argument('--version', default=None)
+    parser.add_argument('--disable-notification', default=False)
```

Comment:
```suggestion
    parser.add_argument('--disable-notification', action='store_const', const='True', default='False')
```
avoids the cast, but overall more typing. Up to you

Reply:
agreed.